src/Bridges/Constraint/Constraint.jl

```diff
@@ -22,97 +22,115 @@ for filename in readdir(joinpath(@__DIR__, "bridges"); join = true)
 end
 
 """
-    add_all_bridges(bridged_model, ::Type{T}) where {T}
+    add_all_bridges(model, ::Type{T}) where {T}
 
-Add all bridges defined in the `Bridges.Constraint` submodule to
-`bridged_model`. The coefficient type used is `T`.
-"""
-function add_all_bridges(bridged_model, ::Type{T}) where {T}
-    if T <: AbstractFloat
-        MOI.Bridges.add_bridge(bridged_model, GreaterToIntervalBridge{T})
-        MOI.Bridges.add_bridge(bridged_model, LessToIntervalBridge{T})
-    end
```

Comment:
@blegat: is it intended behaviour that the bridges depend on the order of calls in this function? That is quite a subtle design choice. But I guess it makes sense if there are two bridges with the same cost.

Reply:
Yes it's quite subtle. In `test/Bridges` we have a few tests verifying that for specific model, we have the right bridges that are applied and these may not pass with any order of addition of bridges

Reply:
Can we tweak the bridge weights instead?